messages/index.js

```diff
@@ -698,6 +725,7 @@ module.exports = {
     cs: 'musí obsahovat položku vyhovující regulárnímu výrazu "{{=e.params.missingPattern}}"',
     de: 'muss ein Attribut nach folgendem Muster haben "{{=e.params.missingPattern}}"',
     es: 'la propiedad debe coincidir con el patrón "{{=e.params.missingPattern}}"',
+    fi: 'ominaisuuden on vastattava muotoa "{{=e.params.missingPattern}}"',
```

Comment:
I don't think it's correct - it looks like "property mush match format" while the message should be "must have property matching pattern". Google suggests "kuvio" for pattern, although I am not sure this is a correct term for regular expression. In any case it should be the same as above.

Reply:
You're absolutely right. "Kuvio" would suit better for textures but in this case I'd go with "kaava" (in this case "kaavaa"), which would translate to "pattern / formula / format".

Changes made to this particular word and phrasing to match original string"